lib/sequelize.js

```diff
@@ -1192,6 +1247,8 @@ Sequelize.prototype.fn = Sequelize.fn;
 Sequelize.prototype.col = Sequelize.col;
 Sequelize.prototype.cast = Sequelize.cast;
 Sequelize.prototype.literal = Sequelize.literal;
+Sequelize.prototype.composition = Sequelize.composition;
```

Comment:
@sushantdhiman do we want to add even more of the aliases?
IIrc the plan was to make people import them directly via `const { literal } = require('sequelize')` in the future since they are effectively pure functions with no side effects.

Reply:
The goal of `sequelize.composition`, `sequelize.slot` is to give users functionality similar to `sequelize.literal`, but with support for binded parameters. The goal of `sequelize.placeholder` is to replace `Op.placeholder` in a later patch, which I think it is unsafe and doesn't work with compositions. So all in all, I think these aliases are necessary.

```
// This is the only example that came to my mind
// I believe it can be used for more powerfull things, like what people used literal for
const field = Model.rawAttributes.field;
Model.findAll({
  where: {
    field: {
      [Op.overlap]: sequelize.composition('int4range(', sequelize.slot(unsafeStart, field), sequelize.slot(unsafeEnd, field), ')')
    }
  },
});
```